sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -3600,8 +3600,10 @@ object GpuOverrides extends Logging {
         TypeSig.STRING,
         Seq(ParamCheck("struct",
           (TypeSig.BOOLEAN + TypeSig.STRING + TypeSig.integral + TypeSig.FLOAT +
+            TypeSig.DECIMAL_64 + TypeSig.DECIMAL_128 +
```

Comment:
DECIMAL_128 implied DECIMAL_64 so it is redundant to have both here.

Reply:
I have removed DECIMAL_64